tests/fast-integration/package-lock.json

```diff
@@ -1,4391 +1,54 @@
 {
   "name": "fast-integration-tests",
   "version": "0.0.1-alpha.12",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Uh! One thing! :D I think we can stick to the second version of the lock file there